doc/ChangeLog.md

```diff
@@ -19,6 +19,8 @@ All notable changes to the project are documented in this file.
 - The timeout before giving up on loading the `startup-config` at boot
   is now 1 minute, just like operations via other front-ends (NETCONF
   and RESTCONF). This was previously (incorrectly) set to 10 seconds.
+- Fix #616: Silent failure when selecting bash as login shell for
+  non-admin user, this silent lock has been removed.
```

Comment:
Nitpick.  Place numbered issues first, in order, followed by other "misc" fixes :)